NetKAN/ContractConfiguratorRO.netkan

```diff
@@ -0,0 +1,11 @@
+{
+    "spec_version": "v1.16",
+    "identifier": "ContractConfiguratorRO",
```

Comment:
```suggestion
    "identifier": "ContractConfigurator-RO",
```

The other RO-specific forked mods have a dash before `-RO`, let's keep this consistent with those.

![image](https://user-images.githubusercontent.com/1559108/174464060-aea14ed6-3aa8-4d1e-94fd-2e4c207f20e5.png)